assets/prototype/domain/eventEditor/containers/useConfig.js

```diff
@@ -0,0 +1,6 @@
+import { useContext } from 'react';
```

Comment:
Didn't made it a `.ts` file cause it would require a chain of other changes.
Will loop back later when will have a better TS coverage.

Reply:
Also in other PR we might unite these kind of hooks in a special folder.